src/homemate_bridge/cli.py

```diff
@@ -229,15 +240,9 @@ def order_state_change(self, new_state):
         self.request.sendall(packet)
 
     def handle(self):
-        # Close the connection if the switch doesn't send anything in 30 minutes
-        # See !1
-        self.request.settimeout(60 * 30)
-
```

Comment:
Why remove this block?

Reply:
I wish I had a good reason.. Adding it back in.